### DISCUSSION TITLE
Security model used in the API

### DISCUSSION THREAD

**Discussion by GregHolroyd** [migueesc123/PowerBIRESTAPI#169] [General]
This Custom Connector is fantastic and so much easier than trying to grapple with the multiples APIs to pull the same data

Appreciate this is still Beta is there any documentation on what the security model it uses as it obviously requires logon with very elevated roles to get the Capacity and downstream data?

Anything which can provide reassurance that the user credentials are purely between PowerBI and Microsoft Azure authentication and nothing else would be useful.

Any possibility of getting this to be a certified connector ?

**Comment by klinejordan** [comment]
You're part of a long list of people who have asked "can you get his connector certified" and I have no ability to answer that. I don't work for Microsoft and as the owner of the data source they'd need to be the one to certify it.

As for security, the implementation just interacts with Azure AD the same way any connector that uses Microsoft Account does: https://learn.microsoft.com/en-us/power-query/handling-authentication#azure-active-directory-authentication
